.github/workflows/build.yml

```
@@ -18,17 +18,18 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [8.0, 8.1]
-                node: [10.x]
-                mysql: [5.7, 8.0]
-                symfony: [^4.4, ~5.2.0, ~5.3.0]
-                sylius: [~1.10.0, ~1.11.0]
+                php: ["8.0", "8.1"]
+                symfony: ["^5.2"]
```

```suggestion
                symfony: ["^5.4"]
```
as Sylius 1.11 and 1.12 does not support 5.2 and 5.3 SF versions.

---

https://github.com/Sylius/ShopApiPlugin/pull/738